specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2018-09-01/containerregistry_build.json

```diff
@@ -779,6 +779,10 @@
             "type": "string"
           }
         },
+        "runErrorMessage": {
```

Comment:
PR titles indicates this is read-only.  If this is the case then I'd suggest adding `"readOnly": true` to this definition.